metadata-ingestion/src/datahub/sdk/lineage_client.py

```diff
@@ -260,7 +250,9 @@ def add_dataset_lineage_from_sql(
         schema_aware: bool = True,
     ) -> None:
         """Add lineage by parsing a SQL query."""
-        from datahub.sql_parsing.sqlglot_lineage import create_lineage_sql_parsed_result
+        from datahub.sql_parsing.sqlglot_lineage import (
+            create_lineage_sql_parsed_result,
+        )
 
         # Parse the SQL query to extract lineage information
         parsed_result = create_lineage_sql_parsed_result(
```

Comment:
the `or ""` is not necessary

Reply:
also let's not expose the `schema_aware` flag - it should always be true, which is the default